.github/workflows/ci.yml

```diff
@@ -50,11 +50,23 @@ jobs:
           privatePackages/*/lib
           privatePackages/*/tsconfig.tsbuildinfo
           .jest-cache
+          .eslintcache
+          packages/*/.eslintcache
+          utils/*/.eslintcache
```

Comment:
I guess these patterns only match one subdirectory? I think we might add more nesting, so probably better to use `**` instead of `*`

Reply:
I think something like `**/.eslintcache` should work